checkov/terraform/checks/utils/base_cloudsplaining_iam_scanner.py

```diff
@@ -0,0 +1,49 @@
+import json
+import logging
+from abc import abstractmethod
+from typing import Dict, List, Any, Union
+
+from checkov.common.models.enums import CheckResult
+
+from cloudsplaining.scan.policy_document import PolicyDocument  # noqa: TC002
```

Comment:
you can actually move it under `TYPE_CHECKING`, but then you also have to import `annotations`

Reply:
cool, fixing now